app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,10 @@
+Welcome, <%= current_user.username%>
+  <br>
+  <%Tweet.all.each do |tweet| %>
```

Comment:
@johnsontaiwo this should use `@tweets` that you should be defining for this route in the TweetsController. Keep business logic out of the views when possible.